scripts/src/actions/fix-yaml-config.ts

```diff
@@ -1,4 +1,6 @@
 import 'reflect-metadata'
+import {addFileToAllRepos} from './shared/add-file-to-all-repos'
 import {format} from './shared/format'
 
+addFileToAllRepos('.github/workflows/stale.yml', '.github/workflows/stale.yml')
```

Comment:
This will run on every PR thus automating adding `stale.yml` to any new repo that is added to the org. 

`addFileToAllRepos` accepts a list of excluded repos as a third arg if we ever have a repo in which we don't want to have `stale.yml`.

Reply:
https://github.com/libp2p/github-mgmt/pull/50/commits/ae518a3b64caa4055e7853732a3b94abc7b92897 is the result of manually triggering `Fix` workflow (https://github.com/libp2p/github-mgmt/actions/runs/3044056716). I needed to run it manually here because the runs that are automatically triggered on PRs `fix-yaml-config.ts` from `master`.